.github/workflows/ci-cd.yaml

```
@@ -75,7 +75,44 @@ jobs:
           MASKINPORTEN_CLIENT_ID: ${{ secrets.MASKINPORTEN_CLIENT_ID }}
           MASKINPORTEN_JWK: ${{ secrets.MASKINPORTEN_JWK }}
           PLATFORM_BASE_URL: ${{ secrets.PLATFORM_BASE_URL }}
-          LEGACY_BASE_URL: ${{ secrets.LEGACY_BASE_URL }}
+          PLATFORM_SUBSCRIPTION_KEY: ${{ secrets.PLATFORM_SUBSCRIPTION_KEY }}
+          SLACK_URL: ${{ secrets.SLACK_URL }}
+
+  deploy-at22:
+    name: deploy at22
+    runs-on: ubuntu-latest
+    environment: test
+    if: always() && !failure() && !cancelled() 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)